python/tools/keras.py

```diff
@@ -16,6 +16,65 @@
 from tensorflow.keras.preprocessing.sequence import pad_sequences
 
 
+def create_ragged_data(
```

Comment:
@scotthlee I actually figured out how to us tf.data and RaggedTensors here (save for getting resampling to work).